dist/index.html

```diff
@@ -1,28 +1,49 @@
 <!doctype html>
 <html>
+
   <head>
-    <link rel="shortcut icon" href="/favicon.ico"/>
+    <link rel="shortcut icon" href="/favicon.ico" />
     <link rel="stylesheet" href="/style.css">
-  </head>
+  <link href="react-uploader.css" rel="stylesheet"></head>
+
   <body>
     <div class="container" style="max-width: 800px; margin: 0 auto">
-
-      <h1>Test this outasldhashjdjksahk</h1>
-      <uploader></uploader>
+      <h1>React Uploader</h1>
+      <uploader test="something"></uploader>
     </div>
 
+    <!-- include UploadCare and ReactUploader as script tag in project -->
     <script charset="utf-8" src="//ucarecdn.com/widget/2.8.2/uploadcare/uploadcare.full.min.js"></script>
     <script src="/react-uploader.js"></script>
-    <script>
-    //  var mockData = [{"cdnUrl":"https://awaw-applications.s3.amazonaws.com/a0f11000003HfzwAAC/artwork/portrait.jpg","artworkTitle":"aaa","artworkYear":"bbb","artworkMedia":"ccc","artworkDimensions":"ddd","description":"eee"},{"cdnUrl":"https://awaw-applications.s3.amazonaws.com/a0f11000003HfzwAAC/artwork/South-America-Patagonia-1-guanaco.jpg"},{"cdnUrl":"https://awaw-applications.s3.amazonaws.com/a0f11000003HfzwAAC/artwork/Torres-del-Paine-Photo-by-jakub-polomski-Latin-America-For-Less.jpg"}]
-
-      var mockData = [];
-      var uploader = new ReactUploader.default(mockData)
 
+    <script>
+      // include config vars for UploadCare
+      UPLOADCARE_LIVE = false;
       UPLOADCARE_LOCALE = "en";
       UPLOADCARE_TABS = "file url facebook gdrive dropbox instagram evernote flickr skydrive";
       UPLOADCARE_PUBLIC_KEY = "38a39e1da7c78a598632";
```

Comment:
Normally id want to make sure we remove it but its a public key anyways so its fine to leave this in there, a lot of the confusion with this project has been how you initialize it properly.

Would you be willing to add more details about all that to the readme?